### PR TITLE
BZ2005326: 4.9 updating quay.io to registry.redhat.io 

### PR DESCRIPTION
**Pull request by kquinn1204**
https://bugzilla.redhat.com/show_bug.cgi?id=2005326

Preview link is: https://deploy-preview-39234--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-create-performance-profiles.html

Updating https://docs.openshift.com/container-platform/4.9/scalability_and_performance/cnf-create-performance-profiles.html removing refs to quay.io and replacing with official registry.redhat.io as per https://bugzilla.redhat.com/show_bug.cgi?id=2005326

### CODE REVIEW

**Comment by kquinn1204** [comment]
@mrniranjan I would be grateful if you could review the updated content. I have tried to verify, but I think something is messed up with my must-gather output.

File: modules/cnf-running-the-performance-creator-profile.adoc [hunk 1]
```diff
@@ -145,6 +158,11 @@ spec:
 
 . Apply the generated profile:
 
+[NOTE]
+====
+Install the Performance Addon Operator before applying the profile. For install instructions, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#installing-the-performance-addon-operator[Installing the Performance Addon Operator]
```
Comment (aireilly) [inline modules/cnf-running-the-performance-creator-profile.adoc:163]:
```suggestion
Install the Performance Addon Operator before applying the profile. For install instructions, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#installing-the-performance-addon-operator_cnf-master[Installing the Performance Addon Operator]
```

**Comment by kquinn1204** [comment]
@mrniranjan I added that note can I get a LGTM

**Comment by mrniranjan** [comment]
LGTM

**Review by lpettyjo — COMMENTED** [review]
LGTM

**Pull request merged**